pkg/kubelet/stats/cri_stats_provider.go

```diff
@@ -449,7 +450,9 @@ func (p *criStatsProvider) getFsInfo(fsID *runtimeapi.FilesystemIdentifier) *cad
 	fsInfo, err := p.cadvisor.GetDirFsInfo(mountpoint)
 	if err != nil {
 		msg := "Failed to get the info of the filesystem with mountpoint"
-		if err == cadvisorfs.ErrNoSuchDevice {
+		if err == cadvisorfs.ErrNoSuchDevice ||
+			errors.Is(err, cadvisorfs.ErrDeviceNotInPartitionsMap) ||
+			errors.Is(err, cadvisormemory.ErrDataNotFound) {
 			klog.V(2).InfoS(msg, "mountpoint", mountpoint, "err", err)
 		} else {
```

Comment:
not to this PR. If this is used by GC, we will return no error in this case. So GC will assume 0 capacity and will fail with the message that there is a zero capacity instead of the error that we can return from here. Would it make sense to bubble this error up?

Reply:
Hm, I can follow-up on this in another PR. I think this would require more intense testing.

Reply:
ack on another follow up sascha. let me land this one